script.js

```diff
@@ -0,0 +1,116 @@
+//interactive navigation menu
+document.addEventListener('scroll', onScroll);
+
+function onScroll(event) {
+  const curPos = window.scrollY;
+  const ancher_block = document.querySelectorAll('#body>div, section');
+  const links = document.querySelectorAll('#menu a');
+  const linksBurger = document.querySelectorAll('#menu-burger a');
```

Comment:
You will select this node element on each scroll event. Maybe you could select it only once and then just use?

Reply:
DONE!